metadata-ingestion/tests/unit/test_glue_source.py

```diff
@@ -266,8 +266,8 @@ def test_platform_config():
 @pytest.mark.parametrize(
     "ignore_resource_links, all_databases_and_tables_result",
     [
-        (True, ({}, [])),
-        (False, ({"test-database": resource_link_database}, target_database_tables)),
+        (True, ([], [])),
+        (False, ([resource_link_database], target_database_tables)),
```

Comment:
Can you please update test for this scenario ? Maybe change `"Name"` for `resource_link_database` to something other than `"test-database"` and verify if database name is replaced ?